app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int = 0) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
```

Comment:
other can be int and float too. Fix typehints everywhere, please
```suggestion
    def __add__(self, other: int | float | "Distance") -> "Distance":
```